packages/wrangler/src/dev.tsx

```diff
@@ -512,34 +518,33 @@ export async function startDev(args: StartDevOptions) {
 		}
 		if (args.experimentalPublic) {
 			throw new UserError(
-				"The --experimental-public field has been renamed to --assets"
+				"The --experimental-public field has been deprecated, try --assets instead."
```

Comment:
```suggestion
				"The --experimental-public field has been deprecated, try --legacy-assets instead."
```

Reply:
because I'm guessing that from this point on we want to direct users towards using `--legacy-assets`